robot.py

```diff
@@ -39,6 +40,7 @@ class MyRobot(magicbot.MagicRobot):
     spinner_controller: SpinnerController
 
     # List components (which represent physical subsystems) here.
+    target_estimator: TargetEstimator
```

Comment:
We probably actually want this to execute before the shoot controller so it can use the latest information, right?